packages/cli/generators/example/downloader.js

```diff
@@ -8,7 +8,7 @@
 const path = require('path');
 const pacote = require('pacote');
 
-module.exports = function downloadAndExtractExample(exampleName, cwd) {
+module.exports = async function downloadAndExtractExample(exampleName, cwd) {
```

Comment:
Ouch! I was incorrectly using `await` in a function that was not marked as `async` before. This is not valid a JavaScript, although it really makes me wonder how is it possible that are tests are still passing.

Reply:
Interesting ... does it mean the test is possibly incorrect?

Reply:
Technically, this source code is an invalid JavaScript that should have produced parsing error when loaded by V8. My guess is that V8 is perhaps clever enough to detect presence of `await` keyword and mark the function as `async` automatically? IDK.